spec/requests/lots_request_spec.rb

```diff
@@ -1,4 +1,89 @@
 require 'rails_helper'
 
-RSpec.describe 'Lots', type: :controller do
+RSpec.describe LotsController, type: :controller do
+  describe 'GET #index' do
+    subject(:make_request) { get :index }
+
+    context 'when user is signed in' do
+      include_context 'with current user'
+
+      it_behaves_like 'response with code', code: 200, request_required: true
+    end
+
+    context 'when user is NOT signed in' do
+      include_context 'with current user', signed_in: false
+
+      it_behaves_like 'response with code', code: 200, request_required: true
+    end
```

Comment:
There's no point to include `with current user` context in this case. When user is not signed in, there's no any current user.

```suggestion
    context 'when user is NOT signed in' do
      it_behaves_like 'response with code', code: 200, request_required: true
    end
```